src/main/java/io/dashboardhub/pipelinedashboard/config/SecurityConfig.java

```diff
@@ -18,6 +19,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
+@Profile("!test")
```

Comment:
I think so, because we don't want this one to run during `test` mode. Unless you see a better way?